hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/AutoCreationLogic.java

```diff
@@ -196,41 +198,49 @@ public Pair<ResponseCodeEnum, Long> create(
                     "Cannot auto-create an account from unaliased change " + change);
         }
 
-        // checks tokenAliasMap if the change consists an alias that is already used in previous
-        // iteration of the token transfer list. This map is used to count number of
-        // maxAutoAssociations needed on auto created account
-        analyzeTokenTransferCreations(changes);
+        TransactionBody.Builder syntheticCreation;
+        String memo;
+        HederaAccountCustomizer customizer = new HederaAccountCustomizer();
+        if (properties.isLazyCreationEnabled() && alias.size() == EntityIdUtils.EVM_ADDRESS_SIZE) {
+            syntheticCreation = syntheticTxnFactory.createHollowAccount(alias, 0L);
+            memo = LAZY_MEMO;
+        } else {
+            // checks tokenAliasMap if the change consists an alias that is already used in previous
+            // iteration of the token transfer list. This map is used to count number of
+            // maxAutoAssociations needed on auto created account
+            analyzeTokenTransferCreations(changes);
```

Comment:
SHould this logic also apply to lazy creates?
If not why not.
If it's a clean addition we should cover it.
If not we can leave token  transfer for later

Reply:
I propose to do this in a separate PR that is for token tranfers, see https://github.com/hashgraph/hedera-services/pull/4004#issuecomment-1283954379.